docs/unified/protectli/recovery.md

```diff
@@ -256,3 +256,73 @@ and Dasharo open-source firmware.
         ```bash
         flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -c "MX25L12835F/MX25L12845E/MX25L12865E" -w [path_to_binary]
         ```
+
+=== "VP2430"
+
+    ## Prerequisites
+
+    * [Prepared RTE](../../transparent-validation/rte/v1.1.0/quick-start-guide.md)
+    * 6x female-female wire cables
+
+    ## Connections
+
+    To prepare the stand for flashing follow the steps described below:
+
+    1. Open the platform cover.
+    1. Connect the J1 and J2 flash headers to the [SPI
+    header](../../transparent-validation/rte/v1.1.0/specification.md) on RTE.
+
+        | SPI header | VP2430 J2    |
+        |:----------:|:------------:|
+        | Vcc        | pin 1 (Vcc)  |
+        | SCLK       | pin 3 (CLK)  |
+        | MOSI       | pin 4 (MOSI) |
+
+        | SPI header | VP2430 J1    |
+        |:----------:|:------------:|
+        | GND        | pin 4 (GND)  |
+        | CS         | pin 1 (CS)   |
+        | MISO       | pin 2 (MISO) |
+
+    ## Firmware flashing
+
+    To flash firmware follow the steps described below:
+
+    1. Login to RTE via `ssh` or `minicom`.
+    1. Turn on the platform by connecting the power supply.
+    1. Wait at least 5 seconds.
+    1. Turn off the platform by using the power button.
+    1. Wait at least 3 seconds.
+    1. Set the proper state of the SPI by using the following commands on RTE:
+
+        ```bash
+        # set SPI Vcc to 3.3V
+        echo 1 > /sys/class/gpio/gpio405/value
+        # SPI Vcc on
+        echo 1 > /sys/class/gpio/gpio406/value
+        # SPI lines ON
+        echo 1 > /sys/class/gpio/gpio404/value
+        ```
+
+    1. Wait at least 2 seconds.
+    1. Disconnect the power supply from the platform.
+    1. Wait at least 2 seconds.
+    1. Flash the platform by using the following command:
+
+        ```bash
+        flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w [path_to_binary]
```

Comment:
Does it not require passing the chip name? I think it does: https://github.com/Dasharo/osfv-scripts/pull/72/files#diff-b1cf3f5b43eab7daf97bf7dcb6fe3e98369d970ce9ca1adfc99582bca69bb531R3

Reply:
[Fixed](https://github.com/Dasharo/docs/pull/932/files#diff-e41ecb94e6a5ed7028b7ace2692886912e5ac7c09053e5cf5ff6ab3e3315733fR314)